crates/core_arch/src/aarch64/crc.rs

```diff
@@ -13,6 +13,7 @@ use stdarch_test::assert_instr;
 #[inline]
 #[target_feature(enable = "crc")]
 #[cfg_attr(test, assert_instr(crc32x))]
+#[stable(feature = "neon_intrinsics", since = "1.59.0")]
 pub unsafe fn __crc32d(crc: u32, data: u64) -> u32 {
```

Comment:
The FCP only covers the NEON (SIMD) intrinsics, not crc intrinsics.

Reply:
OK. Unmarked it.